docs/DeveloperGuide.md

```diff
@@ -7,26 +7,42 @@ title: Developer Guide
 
 --------------------------------------------------------------------------------------------------------------------
 
-## **Acknowledgements**
+## **Acknowledgements** (OUTDATED)
 
 * {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
 --------------------------------------------------------------------------------------------------------------------
 
-## **Setting up, getting started**
+## **Setting up, getting started** (OUTDATED)
 
 Refer to the guide [_Setting up and getting started_](SettingUp.md).
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## **Design**
 
+RC4HDB has the idea of providing a flexible product which functions well, yet not too complex. Keeping this in mind, 
+the developers are pursuing an iterative approach, adding new features and functionalities amidst the evolving requirements.
```

Comment:
Perhaps we could change the developers to **we** so that it has more of a personal feel to it. Both **we** and **the developers** have different benefits to them, so I'm actually fine with either.

Reply:
I agree with Jordan, but I'm leaning towards the usage of 'we'

Reply:
Yep this would make more sense than just "developers"